cmd/backup/encrypt_archive.go

```diff
@@ -4,20 +4,87 @@
 package main
 
 import (
+	"bytes"
+	"errors"
 	"fmt"
 	"io"
 	"os"
 	"path"
 
+	"github.com/ProtonMail/go-crypto/openpgp/armor"
 	openpgp "github.com/ProtonMail/go-crypto/openpgp/v2"
 	"github.com/offen/docker-volume-backup/internal/errwrap"
 )
 
-// encryptArchive encrypts the backup file using PGP and the configured passphrase.
-// In case no passphrase is given it returns early, leaving the backup file
+func readArmoredKeys(data []byte) (openpgp.EntityList, error) {
```

Comment:
I'm not entirely sure this function is needed. It could be refactored to 
```go
func readArmoredKeys(data []byte) (openpgp.EntityList, error) {
    return openpgp.ReadArmoredKeyRing(bytes.NewReader(data))
}
```

which means we could also just call this inline further down?

Reply:
yeah, you are right. i probably overlooked while refactoring, I will fix this

Reply:
fixed in [3b96590](https://github.com/offen/docker-volume-backup/pull/456/commits/3b96590c9880cb4c38a9201551197ecea1522dc0)